### PR TITLE
more info about not skipping release in upgrade-https://issues.redhat…

### PR DESCRIPTION
**Pull request by swopebe**
….com/browse/ACM-11547

### CODE REVIEW

**Review by dislbenn — APPROVED** [review]
LGTM

File: install/upgrade_hub.adoc [hunk 1]
```diff
@@ -17,7 +19,6 @@ You control your {product-title} upgrades by using the operator subscription set
 You also use these settings when you upgrade to the latest version of {product-title-short} by using the operator. Complete the following steps to upgrade your operator:
 
 **Important:** You cannot revert back to an earlier version after upgrading to a later version in the channel selection. You must uninstall the operator and reinstall it with the earlier version to use a previous version.
```
Comment (dockerymick) [inline install/upgrade_hub.adoc]:
```suggestion
*Important:* You cannot revert back to an earlier version after upgrading to a later version in the channel selection. You must uninstall the operator and reinstall it with the earlier version to use a previous version.
```

**Review by dockerymick — APPROVED** [review]
Small comment, otherwise this looks good to me!

**Review by jc-berger — APPROVED** [review]
/lgtm

**Pull request merged**